Servers/UI/OJS.Servers.Ui/ClientApp/src/components/contests/submission-box/SubmissionBox.tsx

```diff
@@ -319,6 +322,10 @@ const SubmissionBox = () => {
         [ handleCodeChanged, selectedSubmissionType, submissionCode, currentProblem, setInvalidExtensionError ],
     );
 
+    const redirectToAdministration = (url: string) => {
+        window.location.href = url;
```

Comment:
There is a `RedirectExternal.ts` file. You can use it.

Reply:
It cannot be used, since it is not accepting "?" in the url

Reply:
It will require new component, do we need whole component just for one redirect ?